assets/js/script.js

```diff
@@ -22,14 +24,63 @@ const topFunction = () => {
   document.documentElement.scrollTop = 0;
 };
 
-/* menu-mobile */
-// const menuToggle = document.getElementById('menuToggle');
+// 2. Mobile Menu
 const menuWrapper = document.getElementById('menuWrapper');
 
 const menuMob = () => {
   menuWrapper.classList.toggle('open-menu');
 };
 
-const myFunc = menuMob;
-const myFunc2 = topFunction;
-console.log(myFunc, myFunc2);
\ No newline at end of file
+// 3. Project Details PopUp
+const getDetailsList = fetch('./assets/data/data.json').then((result) => result.json());
+
+const pdContainer = document.getElementById('pdContainer');
+const pdTitle = document.getElementById('pdTitle');
+const pdCompany = document.getElementById('pdCompany');
+const pdRole = document.getElementById('pdRole');
+const pdYear = document.getElementById('pdYear');
+const pdImage = document.getElementById('pdImage');
+const pdDescription = document.getElementById('pdDescription');
+const pdTechnologies = document.getElementById('pdTechnologies');
+const pdLive = document.getElementById('pdLive');
+const pdSource = document.getElementById('pdSource');
+
+const hasClass = (element, className) => (` ${element.className} `).indexOf(` ${className} `) > -1;
+
+const renderDetails = (title, company, role, year, imageName, description, technologies, live, github) => {
+  pdTitle.innerHTML = title;
+  pdCompany.innerHTML = company;
+  pdRole.innerHTML = role;
+  pdYear.innerHTML = year;
+  pdImage.src = `./assets/img/work-details/${imageName}`;
+  pdLive.setAttribute('href', live);
+  pdSource.setAttribute('href', github);
+  pdDescription.innerHTML = description;
+
+  for (let i = 0; i < pdTechnologies.children.length; i++) {
+    pdTechnologies.children[i].innerHTML = technologies[i];
+  }
+};
+
+const openDetails = (id) => {
+  if (!hasClass(pdContainer, 'show')) {
+    pdContainer.classList.add('show');
+  }
```

Comment:
- [ ] I noticed that every time I click on the `see project` button it only displays the information of the first popup instead of the one concerning it, kindly fix this and make sure each button displays the corresponding popup window